sync-core/src/test/java/com/cloudant/sync/replication/ReplicationTestBase.java

```diff
@@ -37,6 +37,8 @@
     protected CouchClientWrapper remoteDb = null;
     protected CouchClient couchClient = null;
 
+    private long randomDBSuffix = System.currentTimeMillis();
```

Comment:
After talking with @tomblench we came to conclusion that the system time was unlikely to be the same for the tests running in parallel in jenkins.